db/migrations/20220124131743_user_privileges.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public.user_privileges
+(
+    user_id integer NOT NULL,
+    privilege_id integer NOT NULL,
+    id integer NOT NULL DEFAULT nextval('user_privileges_user_privilege_id_seq'::regclass),
```

Comment:
Check if below works, if so - replace it

```suggestion
    id integer SERIAL,
```